src/Profile/Magento/Converter/MagentoConverter.php

```diff
@@ -94,7 +94,19 @@ protected function getAttributes(array $attributes, string $entityName, string $
             if (in_array($attribute['attribute_code'], $blacklist, true)) {
                 continue;
             }
-            $result['migration_' . $connectionName . '_' . $entityName . '_' . $attribute['attribute_id']] = $attribute['value'];
+            if(array_key_exists('frontend_input', $attribute)) {
```

Comment:
Maybe you could do it in an easier way like this:

```
            $value = $attribute['value'];
            if(isset($attribute['frontend_input']) && $attribute['frontend_input'] === 'boolean') {
                $value = (bool) $value;
            }

            $result['migration_' . $connectionName . '_' . $entityName . '_' . $attribute['attribute_id']] = $value;
```
Then its easier to read and you do not need the else-case and the switch-expression.

Reply:
Yeah. I changed this. I just did the switch in case there are errors on other attribute types as well. Missed to refactor this before doing the pr.